recipes/deep-labelprop/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "deep-labelprop" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deep-labelprop-{{ version }}.tar.gz
+  sha256: b50821fbf60bdc25e885b1ed38c89d046166d22a3bd2a1377b9b443a7afba24d
+
+build:
+  entry_points:
+    - labelprop = labelprop.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - flask ==2.1.0
+    - kornia ==0.6.12
+    - lightning
+    - monai ==0.8.1
+    - nibabel ==3.2.1
+    - numpy
+    - plotext ==4.2.0
+    - python >=3.8
```

Comment:
@nathandecaux is there a reason for the exact pinnings. This will most likely result in solver issues.

Could all the `==` be changed to `>=`?

Reply:
Hi. Thanks for your work. You can add me as the maintainer of the conda package. 
Concerning the licence, I don't know which kind is used by napari, but by default I would say GPL2-or-later. 

As far as the python version is concerned, I guess it doesn't have too much impact, as long as it stays > 3.8.

Reply:
@nathandecaux actually monai needs to be `<1`, since `compute_meandice` has been deprecated and removed in favor of `compute_dice`

Reply:
Corrected in labelprop.